Gemfile

```diff
@@ -4,12 +4,21 @@ ruby '2.3.0'
 
 # gem "rails"
 
+gem 'faraday'
+
 group :development do
   gem 'rake'
 end
 
+group :development, :test do
+  gem 'pry-byebug'
+end
+
 group :test do
+  gem 'rspec'
```

Comment:
Can we keep the dependencies in alphabetical order? It keeps things easier to read when our dependencies grow!